third_party/gpus/rocm/BUILD.tpl

```diff
@@ -180,6 +180,24 @@ cc_library(
     ],
 )
 
+# NOTE(mrodden): used by jax_rocm_plugin to minimally link to hip runtime
```

Comment:
`NOTE(username)` is not commonly used in the XLA codebase (unlike TODO). I'll remove this on import.